### DISCUSSION TITLE
[Project Beta] Drop the Save & Edit buttons where possible.

### DISCUSSION THREAD

**Discussion by jurijsk** [community/community#9447] [Projects and Issues]
Will I be wrong to suggest that people write description or change the state of views to get them saved. Can we assume that? If so, it would be great if everything is saved and editable by default. Like in Google docs and MS word. 

Wouldn't it be great?

**Comment by lisa-wolfgang** [comment]
I agree with OP -- it feels much more intuitive to have column layouts and orderings saved automatically. On the other hand, I do see the value in allowing people to open up the project and make temporary changes to the filtering/sorting of tasks so they can get the information they need more quickly.
To stay consistent with the rest of GitHub, perhaps there should be an `Edit` button next to the title that would more obviously make the page enter an editing context instead of a relatively hidden `Save changes` button. Right now, it feels like some weird hybrid between view-only and permanent edit mode.

**Reply by Zackysh** [reply]
Yeah I come here looking to provide feedback about this "possible" issue. There's already people that didn't found the "save button" and think that any changes should have been saved automatically.
One conservative solution as @lisa-wolfgang says could be adding an "edit" button.
Other intersting one would be to automatically save changes on table/board change or other appropiate moment.

**Reply by lisa-wolfgang** [reply]
Well, we do want to keep in mind that managers or non-devs (who may be less tech-savvy) might want to apply a filter or rearrange some things without saving those changes permanently. Devs would also appreciate not having to worry about others constantly changing their board layouts by accident.

**Reply by Zackysh** [reply]
I agree, maybe the option of an edit button is more suitable for teams, also It can be useful for single-person repos.
Editing in projects is slightly different than editing issues, pull requests ...

**Reply by jurijsk** [reply]
Maybe it makes sense to split the issue into two:
**Editing the project structure**: It makes sense for owner to define structure and keep it stable. But then for users of the project it would make sense preserve last good state without additional actions (i.e save).

**Content editing:** meaning editing of title, descriptions, labels etc. I think it would be very beneficial for users to have "auto-save" behavior/mentality. Think GDocs, Figma, Asana. Again, why do people edit content? The answer in 99% cases is to have it saved and visible to others.

**Reply by lisa-wolfgang** [reply]
If I'm not mistaken, content fields are already saved automatically. It's only the actual structure of the board that needs to be manually saved.

**Reply by jurijsk** [reply]
No it is not unfortunately.
![image](https://user-images.githubusercontent.com/351671/148559782-59a8e7e4-d837-40db-84bf-7a65eae3387a.png)
